runtime/src/in_mem_accounts_index.rs

```diff
@@ -26,8 +26,17 @@ impl<V: IsCached> InMemAccountsIndex<V> {
         self.map.entry(pubkey)
     }
 
-    pub fn items(&self) -> Vec<(K, AccountMapEntry<V>)> {
-        self.map.iter().map(|(k, v)| (*k, v.clone())).collect()
+    pub fn items<R>(&self, range: &Option<&R>) -> Vec<(K, AccountMapEntry<V>)>
+    where
+        R: RangeBounds<Pubkey> + std::fmt::Debug,
```

Comment:
What is the `fmt::Debug` used for here (and same for the other fn)?

Reply:
so it is possible to print/log it. I have found this convenient to have present while debugging and it has no penalty, so I went ahead and added it now.